### ISSUE TITLE
HTTP errors

### ISSUE THREAD

**Issue by lukevers** [alexander-bauer/distru#66]
Not exactly sure what's going on, but here's what happens when I start distru up.

``` bash
2012/11/26 14:23:11 Distru version 0.9.0
2012/11/26 14:23:11 Configuration status:
    Generated in:    0.8.3 
    Indexers:    2 
    AutoIndexing:    5 
    Resouces:    3 
    Sites indexed:   0
2012/11/26 14:23:11 indexer> adding "mesh.lukevers.com"
2012/11/26 14:23:11 indexer> adding "[fcdf:db8b:fbf5:d3d7:64a:5aa3:f326:149c]"
2012/11/26 14:23:11 Started server on port 9049.
2012/11/26 14:23:11 Started webserver on port 9048.
2012/11/26 14:23:16 <-[fc64:8b7b:4940:646d:c65d:4052:f6f8:8e06]:49192> searching "distru"
2012/11/26 14:23:16 <-[fc64:8b7b:4940:646d:c65d:4052:f6f8:8e06]:49193> searching "distru"
2012/11/26 14:23:16 indexer> added "mesh.lukevers.com"
2012/11/26 14:23:16 indexer> adding "[fc69:fd06:5207:9b23:87a4:3b69:f99f:a5e1]"
2012/11/26 14:23:16 indexer> added "[fcdf:db8b:fbf5:d3d7:64a:5aa3:f326:149c]"
2012/11/26 14:23:16 indexer> adding "cjdns.thefinn93.com"
2012/11/26 14:23:18 <-[fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:53016> searching "distru"
2012/11/26 14:23:18 <-[fc64:8b7b:4940:646d:c65d:4052:f6f8:8e06]:49192> results: 3
2012/11/26 14:23:18 <-[fc64:8b7b:4940:646d:c65d:4052:f6f8:8e06]:49194> searching "distru"
2012/11/26 14:23:19 <-[fc64:8b7b:4940:646d:c65d:4052:f6f8:8e06]:49193> results: 3
2012/11/26 14:23:20 <-[fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:53016> results: 3
2012/11/26 14:23:21 <-[fc64:8b7b:4940:646d:c65d:4052:f6f8:8e06]:49194> results: 3
2012/11/26 14:24:24 http: Accept error: accept tcp [::]:9048: too many open files; retrying in 5ms
2012/11/26 14:24:24 <-[fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:53016> searching ""
2012/11/26 14:24:24 <-[fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:53016> results: 0
2012/11/26 14:24:24 http: panic serving [fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:53016: open ui/search.css: too many open files
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:589 (0x70947)
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/proc.c:1443 (0x15f57)
/Users/node/distru/web.go:32 (0x5f5b)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:703 (0x64790)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:941 (0x655e4)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:669 (0x645a3)
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/proc.c:271 (0x1405d)
2012/11/26 14:24:24 http: Accept error: accept tcp [::]:9048: too many open files; retrying in 10ms
2012/11/26 14:24:34 indexer> failed to add "cjdns.thefinn93.com"
2012/11/26 14:24:34 indexer> adding "uppit.us"
2012/11/26 14:24:34 indexer> failed to add "uppit.us"
```

After the last line I just killed it.

**Comment by lukevers** [comment]
I should also point out that this is being run from the development branch.

**Comment by alexander-bauer** [comment]
Closed by ab7e7097c3758e6b97df7412d0ada0dc8e408db5.
